### PR TITLE
br: fix dockerfile base

### PR DESCRIPTION
**Pull request by Ehco1996**
### What problem does this PR solve?

Fix panic issue caused by missing dependencies when calling br.
```
/ # /var/lib/br-bin/br
Error loading shared library libresolv.so.2: No such file or directory (needed by /var/lib/br-bin/br)
```

### Code changes

- [ ] Has Go code change
- [x] Has CI related scripts change

### Tests

- [x] Unit test 
- [x] E2E test 
- [ ] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [x] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
none
```

### CODE REVIEW

File: images/tidb-backup-manager/Dockerfile [hunk 1]
```diff
@@ -20,9 +20,9 @@ ARG TOOLKIT_PKG_NAME=tidb-community-toolkit-${TOOLKIT_NAME}
 RUN \
   wget -nv https://download.pingcap.org/${TOOLKIT_PKG_NAME}.tar.gz \
   && tar -xzf ${TOOLKIT_PKG_NAME}.tar.gz \
-  && tar -xzf ${TOOLKIT_PKG_NAME}/tidb-lightning-${TOOLKIT_NAME}.tar.gz && mv tidb-lightning /tidb-lightning \
```
Comment (WangLe1321) [inline images/tidb-backup-manager/Dockerfile:23]: why remove `tidb-lightning` and `dumpling`? I remember that `tidb-backup-manager` contains functions of them.

Comment (Ehco1996) [reply images/tidb-backup-manager/Dockerfile:23]: `tar -xzf ${TOOLKIT_PKG_NAME}/tidb-lightning-${TOOLKIT_NAME}.tar.gz ` already unzip `tidb-lightning` to current dir. the old scripts will rasie an error 

```
mv: 'dumpling' and '/dumpling' are the same file
```
see more in  https://github.com/pingcap/tidb-operator/actions/runs/4487161238/jobs/7890311242

**Review by grovecai — APPROVED** [review]

**Pull request merged**